packages/design-system/src/ui-helpers/getModelInstanceTaskToolkit.tsx

```diff
@@ -177,7 +177,12 @@ export const getModelInstanceTaskToolkit = (task: string) => {
     case "TASK_CUSTOM":
       return {
         getIcon: (className?: string) => {
-          return <Icons.TaskCustom className={className ?? "h-4 w-4"} />;
+          return (
+            <ComplicateIcons.TaskCustom
+              className={className ?? "h-4 w-4"}
+              pathColor="stroke-semantic-fg-primary"
+            />
+          );
```

Comment:
@orangecoloured We can use cn and let the consumer decide how to override the default style `h-4 w-4`